src/_core/constants/appStrings.js

```diff
@@ -99,18 +99,43 @@ export const PROJECTIONS = {
         code: "EPSG:3413",
         proj4Def:
             "+proj=stere +lat_0=90 +lat_ts=70 +lon_0=-45 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs",
-        extent: [-4194304, -4194304, 4194304, 4194304]
+        extent: [-4194304, -4194304, 4194304, 4194304],
+        aliases: ["urn:ogc:def:crs:EPSG::3413"]
+    },
+    southpolar: {
+        code: "EPSG:3031",
+        proj4Def:
+            "+proj=stere +lat_0=-90 +lat_ts=-71 +lon_0=0 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs",
+        extent: [-4194304, -4194304, 4194304, 4194304],
+        aliases: ["urn:ogc:def:crs:EPSG::3031"]
```

Comment:
I'm guessing this is what you see in the gibs capabilities xml?

Reply:
that is correct